dstep/translator/MacroDefinition.d

```diff
@@ -522,7 +522,7 @@ bool translateFunctAlias(
     {
         Identifier* ident = expr.expr.peek!Identifier;
 
-        if (ident.spelling == "assert")
+        if (ident !is null && ident.spelling == "assert")
```

Comment:
There's a null check further down, I suggest removing that and use an early return instead with the end result of only one null check.

Reply:
Good idea. Should be more robust.